modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc

```diff
@@ -27,6 +35,9 @@ image::proxy-deploy-raml-oas-proxy-af2f8.png[]
 
 == See Also
 
+* xref:api-proxy-landing-page.adoc[API Proxy]
 * xref:proxy-advantages.adoc[Reviewing API Proxy Benefits and Impacts]
 * xref:building-https-proxy.adoc[Building an HTTPS API Proxy]
```

Comment:
This should also be added in related links: https://docs.mulesoft.com/apikit/4.x/validate-4-task. It has APIKit documentation on the validation options we are exposing in the proxy.

Reply:
Done! Thanks!